test-count.js

```diff
@@ -0,0 +1,327 @@
+      <th>A</th>
+      <th>T</th>
+      <th>I</th>
+      <th>A</th>
+      <th>T</th>
+      <th>I</th>
+      <th>A</th>
+    </tr>`;
+  for (let i = 0; i < packages.length; i++) {
+    let packageName = packages[i];
+
+    const totalTest = testCount.get(`${packageName}_totalTest`);
+    const manualTest = testCount.get(`${packageName}_manual`) || 0;
+    const todoTest =
+      testCount.get(`${packageName}_totalTest`) -
+        testCount.get(`${packageName}_manual`) || 0;
+    const smokeTest = testCount.get(`${packageName}_smoke`) || 0;
+    const smokeTodoTest = testCount.get(`${packageName}_smoke_todo`) || 0;
+    const smokeAutomated =
+      (testCount.get(`${packageName}_smoke`) || 0) -
+      (testCount.get(`${packageName}_smoke_todo`) || 0);
+    const regressionTest = testCount.get(`${packageName}_regres`) || 0;
+    const regressionTodoTest = testCount.get(`${packageName}_regres_todo`) || 0;
+    const regressionAutomated =
+      (testCount.get(`${packageName}_regres`) || 0) -
+      (testCount.get(`${packageName}_regres_todo`) || 0);
+    const otherTotalTest = testCount.get(`${packageName}_other`) || 0;
+    const otherTodoTest = testCount.get(`${packageName}_other_todo`) || 0;
+    const automationCompletedPercentage =
+      ((smokeAutomated + regressionAutomated + otherTotalTest) /
+        (manualTest + smokeTest + regressionTest + otherTotalTest)) *
+      100;
+
+    htmlContent += `
+      <tr>
+        <td>${packageName}</td>
+        <td>${totalTest}</td>
+        <td>${manualTest}</td>
+        <td>${todoTest}</td>
+        <td>${smokeTest}</td>
+        <td>${smokeTodoTest}</td>
+        <td>${smokeAutomated}</td>
+        <td>${regressionTest}</td>
+        <td>${regressionTodoTest}</td>
+        <td>${regressionAutomated}</td>
+        <td>${otherTotalTest}</td>
+        <td>${otherTodoTest}</td>
+        <td>${otherTotalTest - otherTodoTest}</td>
+        <td>${
+          Math.round((automationCompletedPercentage + Number.EPSILON) * 100) /
+          100
+        }%</td>
+
+      </tr>`;
+
+    jsonContent += `
+            "${packageName}" : {
+                "total" : ${totalTest},    
+                "manual" : ${manualTest},
+                "possible_automation" : ${todoTest},
+                "smoke" : {
+                    "total" : ${smokeTest},
+                    "todo" : ${smokeTodoTest},
+                    "automated" : ${smokeAutomated}
+                },
+                "regression" : {
+                    "total" : ${regressionTest},
+                    "todo" : ${regressionTodoTest},
+                    "automated" : ${regressionAutomated}
+                },
+                "other" : {
+                    "total" : ${otherTotalTest},
+                    "todo" : ${otherTodoTest},
+                    "automated" : ${otherTotalTest - otherTodoTest}
+                },
+                "automation_completed_percentage": ${automationCompletedPercentage}
+            }`;
```

Comment:
Then you can append an object instead of a string to add an element to that object:

```suggestion
    jsonContent[packageName] = {
      total: totalTest,
      manual: manualTest,
      // ....
    };
```